src/CustomModalLayout/index.d.ts

```diff
@@ -8,7 +8,7 @@ export interface CustomModalLayoutProps {
   dataHook?: string;
   theme?: ModalTheme;
   onCloseButtonClick?(): void;
-  title?: string;
+  title?: string | React.ReactNode;
```

Comment:
String is a subtype of a ReactNode
```suggestion
  title?: React.ReactNode;
```